microsoft/testsuites/nvme/nvme.py

```diff
@@ -8,7 +8,27 @@
 from lisa.features import Nvme
 from lisa.sut_orchestrator.azure.platform_ import AzurePlatform
 from lisa.testsuite import simple_requirement
-from lisa.tools import Lscpu
+from lisa.tools import Cat, Fdisk, Lscpu, Mkfsext, Mkfsxfs, Mount, Nvmecli
+from lisa.util import SkippedException
+
+
+def _format_mount_disk(
+    node: Node,
+    namespace: str,
+    file_system: str,
+) -> None:
+    mount_point = namespace.rpartition("/")[-1]
+    fdisk = node.tools[Fdisk]
+    mount = node.tools[Mount]
+    mount.operations(namespace, mount_point, command="umount")
+    fdisk.make_partition(namespace)
+    if "xfs" == file_system:
```

Comment:
If they are always used together with fdisk, the logic can be in fdisk, not in test case. The two tools can be used by fdisk.

Reply:
Avoid to use str on very limited candidates. the `file_system` can be an enum.